### PR TITLE
Add instructions for reporting issues of the JNoSQL databases & JNoSQL extensions in this repository

### LINKED ISSUE #339: Enhance Issues expirience at Eclipse JNoSQL
Hello everyone, we're working to centralize to put the Eclipse JNoSQL issues in a single repository on this current one. 

To make it possible, we have several tasks to do:

- [x] Create "good first issue" label
- [x] Move the issues from [driver](https://github.com/eclipse/jnosql-communication-driver/issues) to JNoSQL
- [x] Move the issues from [mapping](https://github.com/eclipse/jnosql-mapping-extension/issues) to JNoSQL
- [x] Disable driver issues to JNoSQL
- [x] Disable mapping issues to JNoSQL
- [ ] Put on the README documentation how to open an issue on all of three Eclipse JNoSQL repositories.

### PR DESCRIPTION
**Pull request by viniciusbrg**
Closes #339 

- Adds instructions in the readme for users who want to open an issue related to JNoSQL Databases&Extensions
- Updates the feature request & bug report issue templates with a dropdown allowing the user to pick the respective project

image

image

### CODE REVIEW

**Comment by genie-jnosql** [comment]
Can one of the admins verify this patch?

File: README.adoc [hunk 1]
```diff
@@ -624,6 +624,10 @@ Having trouble with Eclipse JNoSQL? We’d love to help!
 
 Please report any bugs, concerns or questions with Eclipse JNoSQL to https://github.com/eclipse/jnosql[https://github.com/eclipse/jnosql].
 
+If your issue refers to the https://github.com/eclipse/jnosql-databases[JNoSQL databases project] or
```
Comment (otaviojava) [inline README.adoc:627]: Great, please, don't forget to add it to the other repositories:
Databases and extentions.

**Review by otaviojava — APPROVED** [review]

**Comment by otaviojava** [comment]
Thank you, merged.

Please, it is essential to refer to the other two repositories.

**Comment by otaviojava** [comment]
Thanks!!

**Pull request merged**